Edits for CMS Chapter (#2566)
HTTPArchive/almanac.httparchive.org: src/content/en/2021/cms.md

Our definition of a CMS in this chapter uses <a hreflang="en" href="https://www.wappalyzer.com/technologies/cms">Wappalyzer's definition</a> of a CMS.

We encourage CMSs to contribute to this <a hreflang="en" href="https://github.com/AliasIO/wappalyzer">open-source project</a> to improve detection and classification in the future.

Shopify, Magento, Webflow, and some other platforms do not appear in this chapter's analysis, because they are not marked as a CMS in Wappalyzer.

Ecommerce platforms make a substantial part of non-CMS sites and are covered in the [Ecommerce chapter](./ecommerce). For example, Shopify grew substantially in the past year and accounted for 3.7% of websites in July according to <a hreflang="en" href="https://w3techs.com/technologies/history_overview/content_management/all/q">W3Techs</a>.

Our research identified over 200 individual CMSs, with these ranging from a single install to millions on a single CMS.

> Some of them are open source (e.g., example, WordPress and Joomla) and some of them are proprietary (e.g., Wix and Squarespace). Some CMS platforms can be used on "free" hosted or self-hosted plans, and there are also options for using these platforms on higher-tiered plans even at the enterprise level.

Reviewer: @tunetheweb `e.g., example` sounds off, was this on purpose?

Reviewer 2: Nope! Was trying to decide between "e.g., Wordpress and Joomla" and "for example, Wordpress and Joomla" (I always think the comma after e.g. looks "off", but apparently that's the correct usage! IT doesn't bother me as much with "for example").

Anyway in my dithering I left a bit of both in! Will remove.

```suggestion
Some of them are open source (e.g., WordPress and Joomla) and some of them are proprietary (e.g., Wix and Squarespace). Some CMS platforms can be used on "free" hosted or self-hosted plans, and there are also options for using these platforms on higher-tiered plans even at the enterprise level.
```

  sheets_gid="362035103",
  sql_file="cms_adoption.sql"
) }}

As of July 2021, over 45% of public websites are powered by a CMS platform, indicating growth of over 7% [from 2020](../2020/cms#cms-adoption). This breaks down to 45% on desktop, up from 42% in 2019, and 46% on mobile, up from 42% in 2020.

It is interesting to compare these numbers with another commonly used dataset, such as <a hreflang="en" href="https://w3techs.com/technologies/history_overview/content_management/all/q">W3Techs</a>, which reported that as of July 2021, 64.6% of websites are created using a CMS, up from 59.2% in July 2020, which is an increase of over 9%.

The deviation between our analysis and W3Techs' analysis can be explained by a difference in research methodologies, and the definition of what is a CMS.

W3Techs definition is the following: "_Content Management Systems are applications for creating and managing the content of a website. We include all such systems in this category, also systems that are often classified as wikis, blog engines, discussion boards, static site generators, website editors or any type of software that provides website content_."

> Wappalyzer has a stricter definition of a <a hreflang="en" href="https://www.wappalyzer.com/technologies/cms">CMS</a>, which excludes some major CMSs which appear in W3Techs reports.

Reviewer 2: ```suggestion
As mentioned previously, Wappalyzer has a stricter definition of a CMS, which excludes some major CMSs which appear in W3Techs reports.
```

> {# TODO You don't explain whats different? I guess it's implied that Wappalyzer has a stricter definition excluding Shopify and the like, but should be be more explicit on this rather than assuming the readers will follow? #}

Reviewer 2: @alonkochba thoughts on this?

Reviewer: Added another sentence, let me know if it's better.
I already expand on this in the "What is a CMS" section, so it felt implied, but now it's a bit more explicit.

Reviewer 2: Thanks. Agree it's basically covered about but think it's better reiterating. Suggested a very slight tweak but resolving this.

WordPress, the most commonly used platform, is used by 33.6% of these websites, up from 31.4% in 2020, a 7% increase in total adoption.

In percentage terms, Joomla and Drupal adoption is dropping–Joomla sites accounted for 1.9% of websites, down from 2.1% last year (9.5% decrease), and Drupal dropped from 2% to 1.8% (10% decrease). Absolute adoption did increase, but as a percentage of both overall CMS usage and of our (ever increasing!) data set, it is smaller.

> {# TODO Added a comment about absolute figures as think it's interesting. Feel free to remove if you disagree. #}

Reviewer 2: @alonkochba you OK with this?

Reviewer: I'm fine with this clarification, though it's a bit hard to determine if absolute adoption grew or not, since the dataset itself is growing every year.

Reviewer 2: Yeah I agree. Just conscious that some of the CMSs might complain with the previous wording and argue differently. Think this clarification covers us a bit should that happen.

Reviewer 2: ```suggestion
In percentage terms, Joomla and Drupal adoption is dropping–Joomla sites accounted for 1.9% of websites, down from 2.1% last year (9.5% decrease), and Drupal dropped from 2% to 1.8% (10% decrease). Absolute adoption did increase, but as a percentage of both overall CMS usage and of our (ever increasing!) data set, it is smaller.
```

  sql_file="top_cms_by_rank.sql"
) }}

3.1% of mobile sites in the top 1K are built with WordPress, and 33.6% of all sites. Drupal maintains a higher adoption rate within the mid-ranged rankings (10K–1M), while most of Wix and Squarespace sites are ranked outside the top 1M sites.

## CMS user experience

{# TODO Headings should always have at least one sentence. Can we add an introduction saying what this section is covering and whu? #}

> ### Core Web Vitals

Reviewer 2: @alonkochba you fill something in here?

Reviewer: Added a short intro

Reviewer 2: LGTM! Thanks for that.
